main.go

```diff
@@ -128,6 +128,7 @@ func generate(in configInput) ([]byte, error) {
 				"TRAFFICDIRECTOR_GCP_PROJECT_NUMBER": strconv.FormatInt(in.gcpProjectNumber, 10),
 			},
 		},
+		GRPCServerResourceNameId: "grpc/server",
```

Comment:
FWIW, I thought this was going to be mirroring the udpa resource name format, with a repeated string field.

Reply:
It was recently changed to be just a string instead of a list of strings. I'll try to look up where this is documented.